python/plugins/db_manager/db_plugins/oracle/info_model.py

```diff
@@ -142,7 +142,7 @@ def generalInfo(self):
                     "DBManagerPlugin", "Rows (estimation):"),
                  self.table.estimatedRowCount)
             )
-        if self.table.rowCount or self.table.rowCount >= 0:
+        if self.table.rowCount is not None and self.table.rowCount >= 0:
```

Comment:
Just replacing `or` with `and` should fix the issue no? the `is not None` check is not necessary. 

this works in python3
```python
a=None
>>> if a and a > 100 :
       print("test")
```

Reply:
Not exactly. Row count can be 0 which gives a False value and will not satisfy the condition. It is 'greater than or equal' to be precise.

Reply:
Indeed I miss the case where rowCount equals 0. Actually, the way you write it removes the ambiguity in both case. Forget about my comments